reference/commands/creator/export-pkg.rst

```diff
@@ -60,127 +62,37 @@ and or '--profile'.
-<existing_binaries>`.
-
-:command:`export-pkg` has two different modes of operation:
-
-- Specifying :command:`--package-folder` will perform a copy of the given folder, without executing the ``package()`` method.
-  Use it if you have already created the package, for example with :command:`conan package` or
-  with ``cmake.install()`` from the ``build()`` step.
-- Specifying :command:`--build-folder` and/or :command:`--source-folder` will execute the ``package()`` method,
-  to filter, select and arrange the layout of the artifacts.
-
-**Examples**:
-
-- Create a package from a directory containing the binaries for Windows/x86/Release:
-
-  Having these files:
-
-  .. code-block:: text
-
-      Release_x86/lib/libmycoollib.a
-      Release_x86/lib/other.a
-      Release_x86/include/mylib.h
-      Release_x86/include/other.h
-
-  Run:
-
-  .. code-block:: bash
-
-      $ conan new Hello/0.1 --bare  # In case you still don't have a recipe for the binaries
-      $ conan export-pkg . Hello/0.1@user/stable -s os=Windows -s arch=x86 -s build_type=Release --build-folder=Release_x86
-
-- Create a package from a user folder build and sources folders:
-
-  Given these files in the current folder
-
-  .. code-block:: text
-
-      sources/include/mylib.h
-      sources/src/file.cpp
-      build/lib/mylib.lib
-      build/lib/mylib.tmp
-      build/file.obj
-
-  And assuming the ``Hello/0.1@user/stable`` recipe has a ``package()`` method like this:
-
-  .. code-block:: python
-
-      def package(self):
-          self.copy("*.h", dst="include", src="include")
-          self.copy("*.lib", dst="lib", keep_path=False)
-
-  Then, the following code will create a package in the conan local cache:
-
-  .. code-block:: bash
-
-      $ conan export-pkg . Hello/0.1@user/stable -pr=myprofile --source-folder=sources --build-folder=build
-
-  And such package will contain just the files:
-
-  .. code-block:: text
-
-      include/mylib.h
-      lib/mylib.lib
-
-- Building a conan package (for architecture x86) in a local directory and then send it to the local cache:
-
-  **conanfile.py**
-
-  .. code-block:: python
-
-      from conans import ConanFile, CMake, tools
-
-      class LibConan(ConanFile):
-          name = "Hello"
-          version = "0.1"
-          ...
+The :command:`export-pkg` command let you create a package from already existing files
+in your workspace, it can be useful if you are using a build process external to Conan
+and do not want to provide it with the recipe. Nevertheless, you should take into
+account that it will generate a package and Conan won't be able to guarantee its
+reproducibility or regenerate it again. This is **not** the normal or recommended flow
+for creating Conan packages.
 
-          def source(self):
-              self.run("git clone https://github.com/memsharded/hello.git")
+Execution of this command will result in several files copied to the ``package_id``
```

Comment:
It is actually the package folder... maybe just "the package folder in the cache"?